pkg/odo/cli/pipelines/init.go

```diff
@@ -69,6 +69,13 @@ func (io *InitParameters) Validate() error {
 	if len(removeEmptyStrings(strings.Split(gr.Path, "/"))) != 2 {
 		return fmt.Errorf("repo must be org/repo: %s", strings.Trim(gr.Path, ".git"))
 	}
+	if io.gitOpsWebhookSecret == "" {
+		gitSecret, err := GenerateSecureString()
+		if err != nil {
+			return err
```

Comment:
This error could be improved...the error you'll get back from `GenerateSecureString` will likely be a technical error, which may not be suitable for users?

Imagine if you ran the tool and it failed with an error that said something like `crypto/rand: blocked for 60 seconds waiting to read random data from the kernel`...huh?!